src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const allMen = people.filter(person => person.sex === 'm');
+  const menOfTheCentury = allMen.filter(person =>
+    Math.ceil(person.died / 100) === century);
+  let men;
+
+  century === undefined ? men = allMen : men = menOfTheCentury;
```

Comment:
так писать в тернарнике нельзя
```suggestion
   cosnt men = century === undefined ? allMen : menOfTheCentury;
```